platform/event/core_event_data.proto

```diff
@@ -0,0 +1,45 @@
+/**
+ * # Core Event Data
+ * Contains information about an event, and its parents.
+ *
+ * ### Keywords
+ * The key words "MUST", "MUST NOT", "REQUIRED", "SHALL", "SHALL NOT",
+ * "SHOULD", "SHOULD NOT", "RECOMMENDED", "MAY", and "OPTIONAL" in this
+ * document are to be interpreted as described in [RFC2119](https://www.ietf.org/rfc/rfc2119).
+ */
+syntax = "proto3";
+
+package proto;
```

Comment:
We need to start using proper packages.
`package com.hedera.hapi.platform.event;`
Reference other protos like C++ namespaces (elide common prefixes).
The "old" messages are all prefixed with `proto` until we're able to start fixing those.